### PR TITLE
[Draft] defer - pretty print stack

### PR DESCRIPTION
**Pull request by JTurcotti**
Replace this paragraph with a description of your changes and rationale. Provide links to external references/discussions if appropriate.

Resolves #NNNNN, fix apple/llvm-project#MMMMM.

### CODE REVIEW

**Comment by JTurcotti** [comment]
@swift-ci please smoke test

File: SwiftCompilerSources/CMakeLists.txt [hunk 1]
```diff
@@ -255,7 +255,7 @@ else()
   # step 2: build a library containing that source file. This library depends on all the included header files.
   #         The swift modules can now depend on that target.
   #         Note that this library is unused, i.e. not linked to anything.   
-  add_library(importedHeaderDependencies "${CMAKE_CURRENT_BINARY_DIR}/HeaderDependencies.cpp")
+  add_library(importedHeaderDependencies "${CMAKE_CURRENT_BINARY_DIR}/HeaderDependencies.cpp" ../include/swift/AST/ActorIsolation.cpp)
```
Comment (eeckstein) [inline SwiftCompilerSources/CMakeLists.txt:258]: What is this change for?

**Review by eeckstein — COMMENTED** [review]
Can you provide some description for the PR?

**Comment by JTurcotti** [comment]
superseded by https://github.com/apple/swift/pull/66929

**Pull request closed without merging**